test/test_image.py

```diff
@@ -22,6 +22,35 @@ def get_images(directory, img_ext):
 
 
 class ImageTester(unittest.TestCase):
+    def test_read_jpeg(self):
+        for img_path in get_images(IMAGE_ROOT, ".jpg"):
+            img_pil = torch.from_numpy(np.array(Image.open(img_path)))
+            img_ljpeg = read_jpeg(img_path)
+
+            norm = img_ljpeg.shape[0] * img_ljpeg.shape[1] * img_ljpeg.shape[2] * 255
+            err = torch.abs(img_ljpeg.flatten().float() - img_pil.flatten().float()).sum().float() / (norm)
+            self.assertLessEqual(err, 1e-2)
```

Comment:
Can't we instead do something like
```python
diff = (img_ljpeg.float() - img_pil.float()).abs().max()
```
and check that the difference is small? I think it would be easier for me to understand the differences that we get.